app/components/sbom/app-logo/index.hbs

```diff
@@ -0,0 +1,7 @@
+<AkStack
+  local-class='app-logo-container'
+  @alignItems='center'
+  @justifyContent='center'
+>
+  <img data-test-sbomApp-logo src={{this.iconUrl}} alt='' role='none' />
```

Comment:
- Empty `alt` attribute. 
- Is the `role='none` required?

Reply:
yeah the logo container is small for an alt text and empty alt requires role='none' (linting issue)